### ISSUE TITLE
No key binding to show/hide the calendar/clock

### ISSUE THREAD

**Issue by alexchandel** [JapaMala/armok-vision#50]
It's somewhat difficult to click the calendar/clock tab, since Armok Vision hides the cursor on the second click. There should be a key binding for this.

**Comment by JapaMala** [comment]
I'll add it in.

**Comment by alexchandel** [comment]
I noticed the P key claims to show/hide the DF time, but it seems to have no effect. Is that for the solar cycle, not the clock?

**Comment by JapaMala** [comment]
That is correct. It works together with the [ and ] keys. I guess that could be made more clear.
